config/puma.rb

```diff
@@ -25,14 +25,14 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 2 }
 
 # Use the `preload_app!` method when specifying a `workers` number.
 # This directive tells Puma to first boot the application and load code
 # before forking the application. This takes advantage of Copy On Write
 # process behavior so workers use less memory.
 #
-# preload_app!
+preload_app!
```

Comment:
Thank you for noticing this! I think Heroku also recommends it?

Reply:
It does! https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#preload-app

Reply:
I only did this because the render docs showed it in their example. I have not actually dug in to see what this does in practice (and I especially didn't dig into Puma yet). But glad it may help sometime! May want to move some of this into a different PR, or repurpose this one as a general spruce-up of deployment stuff.

Reply:
Yea, I think an independent PR to enabling preloading app + workers is a good idea; especially so we can evaluate how it impacts using `binding.pry`/`byebug` in development and test mode.